source/getting-started/installation-raspberry-pi.markdown

```diff
@@ -1,29 +1,207 @@
 comments: false
 sharing: true
 footer: true
 ---
 
-Home Assistant requires the Raspberry Pi to run [Raspbian Jessie](https://www.raspberrypi.org/downloads/raspbian/). This version was released on September 24, 2015 and comes by default with Python 3.4 which is required for Home Assistant.
+### {% linkable_title Installation %}
 
-Execute the following code in a console:
+This installation of Home Assistant requires the Raspberry Pi to run [Raspbian Jessie Lite](https://www.raspberrypi.org/downloads/raspbian/).
+The installation will be installed in a [Virtual Environment](/getting-started/installation-virtualenv) with minimal overhead. Instructions assume this is a new installation.
 
+Connect to the Raspberry Pi over ssh. Default password is `raspberry`.
+```bash
+$ ssh pi@ipadress
+```
+
+Changing the default password is encouraged.
+```bash
+$ passwd
+```
+
+Update the system.
+```bash
+$ sudo apt-get update
+$ sudo apt-get upgrade -y
+```
+
+Install the dependencies.
+```bash
+$ sudo apt-get install python3 python3-venv python3-pip python3-sphinx
+$ sudo pip3 install virtualenv
+```
+
+Add a account for Home Assistant called `homeassistant`.  
+Since this account is only for running Home Assistant the extra arguments of `-rm` is added to create a system account and create a home directory.
+```bash
+$ sudo useradd -rm homeassistant
+```
+
+Next we will create a directory for the installation of Home Assistant and change the owner to the `homeassistant` account.
+```bash
+$ cd /srv
+$ sudo mkdir homeassistant
+$ sudo chown homeassistant:homeassistant homeassistant
+```
+
+Next up is to create and change to a virtual environment for Home Assistant. This will be done as the `homeassistant` account.
+```bash
+$ su -s /bin/bash homeassistant 
+$ cd /srv/homeassistant
+$ virtualenv -p python3 homeassistant_venv
+$ source /srv/homeassistant/homeassistant_venv/bin/activate
+```
+
+Finally the installation of Home Assistant. This will install Home Assistant and any dependencies into the virtual environment.
+```bash
+$ pip3 install homeassistant
+```
+
+Start Home Assistant for the first time. This will complete the installation, create the `.homeasssistant` configuration directory in the `/home/homeassistant` directory and install any basic dependencies.
 ```bash
-$ sudo pip3 install homeassistant
 $ hass
 ```
 
-Running these commands will:
+You can now reach your installation on your raspberry pi over the web interface on [http://ipaddress:8123](http://ipaddress:8123). 
+For instruction on how to configure Home Assistant continue on with [Configuring Home Assistant](/getting-started/configuration/).
+
+### {% linkable_title Raspberry Pi Hardware Specific Components %}
+
+Some components that are specific for the Raspberry Pi can require some further configuration outside of Home Assistant. All commands below are assumbed to be executed with the `pi` account. For full documentation of these components refer to the [components](/components) page.
```

Comment:
In this case yes.  
Since this installation creates a separate account for HA that does not have sudo privileges. Tested it on a friend and he got stuck there since he just continued on from the previous part where you where logged in with the ``homeassistant` account.